examples/react-router/package.json

```diff
@@ -12,7 +12,8 @@
   "dependencies": {
     "fluxible": "^0.5.0",
     "fluxible-addons-react": "^0.1.6",
+    "history": "^1.17.0",
     "react": "^0.13.0",
-    "react-router": "^0.13.1"
+    "react-router": "^1.0.0"
```

Comment:
Good point. 
After I moved the dependencies under examples/package.json, react-router server cannot correctly match the url, and there is no error. I will investigate more later.